_ng/full/generator.js

```diff
@@ -186,6 +187,28 @@ exports.MainGenerator = class MainGenerator {
       this.wrapper.config.save();
   }
 
+  promptTest() {
+    const done = this.wrapper.async();
+
+    let prompts = [
+      {
+        type: "confirm",
+        name: "tests",
+        message: "Do you want to keep test in separate directory?",
+        default: false
```

Comment:
I think the default should be true, since that's the expected (and probably most used) behavior.